src/qute/languageServer/client.ts

```diff
@@ -2,18 +2,17 @@ import * as requirements from './requirements';
 
 import { DidChangeConfigurationNotification, LanguageClientOptions } from 'vscode-languageclient';
 import { LanguageClient } from 'vscode-languageclient/node';
-import { ExtensionContext, commands, workspace, window, ConfigurationTarget, languages } from 'vscode';
+import { ExtensionContext, commands, workspace, window, ConfigurationTarget} from 'vscode';
 import { prepareExecutable } from './quteServerStarter';
 import { registerQuteExecuteWorkspaceCommand, registerVSCodeQuteCommands, synchronizeQuteValidationButton } from '../commands/registerCommands';
 import { QuteClientCommandConstants } from '../commands/commandConstants';
 import { QuteSettings } from './settings';
 import { JavaExtensionAPI } from '../../extension';
-import { QuteInlayHintsProvider } from './inlayHintsProvider';
 
-export function connectToQuteLS(context: ExtensionContext, api: JavaExtensionAPI) {
+export async function connectToQuteLS(context: ExtensionContext, api: JavaExtensionAPI) {
   registerVSCodeQuteCommands(context);
 
-  return requirements.resolveRequirements(api).then(requirements => {
+  return requirements.resolveRequirements(api).then(async requirements => {
```

Comment:
Since this function is now `async`, you can do:

```typescript
import { resolveRequirements } from './requirements';
// ...
export async function connectToQuteLS(context: ExtensionContext, api: JavaExtensionAPI) {
  registerVSCodeQuteCommands(context);
  const requirements = await resolveRequirements(api);
  const clientOptions: LanguageClientOptions = {
  // ...
```

Reply:
Good catch! Fixed.